.github/workflows/build.yml

```diff
@@ -13,21 +13,15 @@ jobs:
       - name: Install deps
         run: |
           yarn install
-      - name: Bootstrap packages
-        run: |
-          yarn lerna bootstrap
-      - name: Force local package symlinks
-        run: |
-          yarn lerna link --force-local
       - name: Verify package dependencies
         run: |
-          yarn check-deps
+          npx depcheck
```

Comment:
Now that we don't have multiple packages in the repo, this is a bit simpler and we don't need the custom script. Figured it makes sense to just use `npx` (they suggest it on their npm docs https://www.npmjs.com/package/depcheck)